<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учет заказов и материалов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="ordersTab">
       <attribute name="title">
        <string>Заказы</string>
       </attribute>
       <layout class="QVBoxLayout" name="ordersLayout">
        <item>
         <widget class="QTableView" name="ordersTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ordersButtonsLayout">
          <item>
           <widget class="QPushButton" name="addOrderButton">
            <property name="text">
             <string>Добавить заказ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggleEditOrdersButton">
            <property name="text">
             <string>Редактировать заказ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteOrderButton">
            <property name="text">
             <string>Удалить заказ</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ClientsTab">
       <attribute name="title">
        <string>Клиенты</string>
       </attribute>
       <layout class="QVBoxLayout" name="ClientsLayout">
        <item>
         <widget class="QTableView" name="ClientsTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ClientsButtonsLayout">
          <item>
           <widget class="QPushButton" name="addClientsButton">
            <property name="text">
             <string>Добавить клиента</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editClientsButton">
            <property name="text">
             <string>Редактировать клиента</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteClientsButton">
            <property name="text">
             <string>Удалить клиента</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TypeOrdersTab">
       <attribute name="title">
        <string>Услуги</string>
       </attribute>
       <layout class="QVBoxLayout" name="TypeOrdersLayout">
        <item>
         <widget class="QTableView" name="TypeOrdersTable">
          <property name="columnCount" stdset="0">
           <number>4</number>
          </property>
          <property name="horizontalHeaderLabels" stdset="0">
           <stringlist>
            <string>ID услуги</string>
            <string>Название</string>
            <string>Материал</string>
            <string>Количество</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="TypeOrdersButtonsLayout">
          <item>
           <widget class="QPushButton" name="addTypeOrdersButton">
            <property name="text">
             <string>Добавить услугу</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toggleEditTypeOrdersButton">
            <property name="text">
             <string>Редактировать услугу</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTypeOrdersButton">
            <property name="text">
             <string>Удалить услугу</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="materialsTab">
       <attribute name="title">
        <string>Материалы</string>
       </attribute>
       <layout class="QVBoxLayout" name="materialsLayout">
        <item>
         <widget class="QTableView" name="materialsTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="materialsButtonsLayout">
          <item>
           <widget class="QPushButton" name="addMaterialButton">
            <property name="text">
             <string>Добавить материал</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editMaterialButton">
            <property name="text">
             <string>Редактировать материал</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteMaterialButton">
            <property name="text">
             <string>Удалить материал</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
